test/unit/middleware/auth/sign_up_with_email_middleware_test.dart

```diff
@@ -32,15 +31,15 @@ void main() {
 
       // setup middleware and invoke the middleware
       final mut = SignUpWithEmailMiddleware(mockAuthService);
-      mut(testStore, SignUpWithEmail(), testDispatcher);
+      mut(fakeStore, SignUpWithEmail(), testDispatcher);
 
       // check that the service's stream emits the expected action and the store
       // dispatched the same action
       mockAuthService
           .emailSignUpStream(testEmail, testPassword)
           .listen(expectAsync1((action) {
             expect(action, equals(testAction));
-            expect(testStore.dispatchedActions.contains(testAction), true);
+            expect(fakeStore.dispatchedActions.contains(testAction), true);
```

Comment:
made a quick pr with `expect(fakeStore.dispatchedActions.contains(testAction), true); ` changed to `expect(fakeStore.dispatchedActions,contains(testAction));` to match how it was suggested to be done in sign_in_with_email_middleware_test.dart

Reply:
thanks for that!